rca/people/models.py

```diff
@@ -656,6 +662,7 @@ class StudentPage(BasePage):
     ]
 
     content_panels = BasePage.content_panels + [
+        FieldPanel("student_user_account"),
```

Comment:
Just to flag that if there are a large number of student accounts the `<select>` could be slow. I'm not sure of the solution, but this might be an option https://github.com/wagtail/wagtail-autocomplete

Reply:
Thanks, that's worth noting, I'll add it to the ticket